doc/languages-frameworks/python.section.md

```diff
@@ -1936,9 +1938,9 @@ Both are also exported in `nix-shell`.
 It is recommended to test packages as part of the build process.
 Source distributions (`sdist`) often include test files, but not always.
 
-The best practice today is to pass a test hook (e.g. pytestCheckHook, unittestCheckHook) into nativeCheckInputs.
-This will reconfigure the checkPhase to make use of that particular test framework.
-Occasionally packages don't make use of a common test framework, which may then require a custom checkPhase.
+The best practice today is to pass a test hook (e.g. pytestCheckHook, unittestCheckHook) into nativeInstallCheckInputs.
+This will reconfigure the installCheckPhase to make use of that particular test framework.
+Occasionally packages don't make use of a common test framework, which may then require a custom installCheckPhase.
```

Comment:
Minor formatting and linking suggestions:
```suggestion
The best practice today is to pass a test hook (e.g. `pytestCheckHook`, `unittestCheckHook`) into [`nativeInstallCheckInputs`](#var-stdenv-nativeInstallCheckInputs).
This will reconfigure the `installCheckPhase` to make use of that particular test framework.
Occasionally packages don't make use of a common test framework, which may then require a custom `installCheckPhase`.
```